spec/support/examples/api.rb

```diff
@@ -21,3 +21,35 @@
     expect(response).to have_http_status(403)
   end
 end
+
+shared_examples 'unprocessable entity' do
```

Comment:
We've a tonne of cases where this test exists, so we'll want to refactor all of those once this is merged.